applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_d_plus_d_minus_damage.h

```diff
@@ -313,6 +313,14 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) GenericSmallStrainDplusDminus
         Matrix& rValue
         ) override;
 
+    /**
+     * @brief If the CL requires to initialize the material response, called by the element in InitializeSolutionStep.
+     */
+    bool RequiresFinalizeMaterialResponse() override
```

Comment:
Not adding the initialize implies it DOES need it, correct?

Reply:
Yes